client/changes.go

```diff
@@ -34,8 +34,8 @@ type Change struct {
 	Ready   bool    `json:"ready"`
 	Err     string  `json:"err,omitempty"`
 
-	SpawnTime time.Time `json:"spawn-time,omitempty"`
-	ReadyTime time.Time `json:"ready-time,omitempty"`
+	SpawnTime time.Time `json:"spawn-time"`
+	ReadyTime time.Time `json:"ready-time"`
```

Comment:
@benhoyt I wonder if `omitzero` was originally intended instead of `omitempty`...

Or perhaps this code was written before Go 1.24, so that was not an option.
I wonder what's the safe thing to do here.

Basically `time.Time` is never "empty", but it can be "zero".
Ref: https://stackoverflow.com/questions/32643815/json-omitempty-with-time-time-field